cmd/clusterawsadm/cmd/bootstrap/iam/cloudformation.go

```diff
@@ -93,7 +93,7 @@ func createCloudFormationStackCmd() *cobra.Command {
 			}
 
 			if err := resolveTemplateRegion(t, cmd); err != nil {
-				return err
+				fmt.Printf("AWS_REGION env not set and --region flag not provided, default configuration will be used\n")
```

Comment:
nit.
fmt.Println("AWS_REGION env not set and --region flag not provided, default configuration will be used")